src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20211207175748_EES2776_AddGeographicLevelToLocation.cs

```diff
@@ -7,7 +7,9 @@ namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations
     public partial class EES2776_AddGeographicLevelToLocation : Migration
     {
         private const string MigrationId = "20211207175748";
-        
+        private const string PreviousLocationTypeMigrationId = "20210719084148";
```

Comment:
If we keep with this pattern of updating Types and Routines, and reference an old one to rollback, we should probably make our `MigrationId` public, and reference the previous class's actual `MigrationId` variable rather than repeat the string here.  That way, we can jump between source more easily.

Reply:
Thanks, good idea! I suppose we can leave the current one `private` until a rollback needs to refer to it.
I have made this change, making the previous migration id constant public, ands then referenced it in these two places.

Reply:
Nice one